ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -180,124 +143,85 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
   };
 
   return (
-    <LoadingSpinnerOverlay
-      isLoading={
-        props.isCgnInfoLoading ||
-        (props.cgnDetails && cardLoading) ||
-        isLoading(props.unsubscriptionStatus)
+    <BonusCardScreenComponent
+      logoUris={logoUris}
+      name={I18n.t("bonus.cgn.name")}
+      organizationName={I18n.t("bonus.cgn.departmentName")}
+      counters={[]}
+      cardBackground={<AnimatedBackground />}
+      status={
+        props.cgnDetails ? <CgnCardStatus card={props.cgnDetails} /> : undefined
+      }
+      cardFooter={
+        <H2
+          style={{
+            textAlign: "center",
+            marginHorizontal: 16,
+            marginBottom: 16
+          }}
+        >
+          {pot.isSome(currentProfile)
+            ? `${currentProfile.value.name} ${currentProfile.value.family_name}`
+            : ""}
+        </H2>
+      }
+      footerCtaPrimary={
+        showDiscoverCta
+          ? {
+              label: getPrimaryactionLabel(canDisplayEycaDetails),
+              onPress: onPressShowCgnDiscounts
+            }
+          : undefined
```

Comment:
Same as above

Reply:
https://github.com/pagopa/io-app/pull/6097#discussion_r1717173575

Reply:
https://github.com/pagopa/io-app/pull/6097#discussion_r1728531606

Reply:
done in 0446780e896c90777b485fee0efd82fef7f3ade2, please check if the change is what you had in mind